.github/workflows/test.yml

```diff
@@ -65,3 +67,14 @@ jobs:
         run: |
           GO111MODULE=on go get github.com/prometheus/prometheus/cmd/promtool@a6be548dbc17780d562a39c0e4bd0bd4c00ad6e2
           make test_promtool
+  stress-test:
```

Comment:
if it is possible to reduce execution time then we should move this test to unit instead of it's own stage